packages/distcc/build.sh

```diff
@@ -0,0 +1,12 @@
+TERMUX_PKG_HOMEPAGE=http://distcc.org/
+TERMUX_PKG_DESCRIPTION="Distributed C/C++ compiler."
+TERMUX_PKG_VERSION=3.2-rc1
+TERMUX_PKG_REVISION=1
+TERMUX_PKG_SHA256=8cf474b9e20f5f3608888c6bff1b5f804a9dfc69ae9704e3d5bdc92f0487760a
+TERMUX_PKG_SRCURL=https://storage.googleapis.com/google-code-archive-downloads/v2/code.google.com/distcc/distcc-3.2rc1.tar.gz
+TERMUX_PKG_BUILD_IN_SRC=yes
+
+termux_step_configure () {
+	#the configure script checks if $PYTHON is executable
+        PYTHON=/etc/hosts sh ./configure -disable-Werror --without-libiberty --prefix=/usr --host `dpkg-architecture -q DEB_HOST_GNU_TYPE`
```

Comment:
On the same theme as above, --prefix=/usr can't be right?

Reply:
The two get combined and the files end up where they should be. I'll still submit a patch using @TERMUX_PREFIX@ hwoever

Reply:
```
shawn@jordon-desktop:~/git/termux-packages$ dpkg-deb -c debs/distcc_3.2-rc1-1_arm.deb 
drwxr-xr-x builder/4139      0 2017-11-27 16:49 ./
drwxr-xr-x builder/4139      0 2017-11-27 16:49 ./data/
drwxr-xr-x builder/4139      0 2017-11-27 16:49 ./data/data/
drwxr-xr-x builder/4139      0 2017-11-27 16:49 ./data/data/com.termux/
drwxr-xr-x builder/4139      0 2017-11-27 16:49 ./data/data/com.termux/files/
drwx------ builder/4139      0 2017-11-27 16:49 ./data/data/com.termux/files/usr/
drwx------ builder/4139      0 2017-11-27 16:49 ./data/data/com.termux/files/usr/etc/
drwx------ builder/4139      0 2017-11-27 16:49 ./data/data/com.termux/files/usr/etc/distcc/
-rw------- builder/4139   1377 2017-11-27 16:49 ./data/data/com.termux/files/usr/etc/distcc/clients.allow
-rw------- builder/4139   1794 2017-11-27 16:49 ./data/data/com.termux/files/usr/etc/distcc/commands.allow.sh
-rw------- builder/4139    203 2017-11-27 16:49 ./data/data/com.termux/files/usr/etc/distcc/hosts
drwx------ builder/4139      0 2017-11-27 16:49 ./data/data/com.termux/files/usr/etc/default/
-rw------- builder/4139     17 2017-11-27 16:49 ./data/data/com.termux/files/usr/etc/default/distcc
drwx------ builder/4139      0 2017-11-27 16:49 ./data/data/com.termux/files/usr/bin/
-rwx------ builder/4139  30892 2017-11-27 16:49 ./data/data/com.termux/files/usr/bin/distccmon-text
-rwx------ builder/4139  71168 2017-11-27 16:49 ./data/data/com.termux/files/usr/bin/distccd
-rwx------ builder/4139  35076 2017-11-27 16:49 ./data/data/com.termux/files/usr/bin/lsdistcc
-rwx------ builder/4139  68116 2017-11-27 16:49 ./data/data/com.termux/files/usr/bin/distcc
drwx------ builder/4139      0 2017-11-27 16:49 ./data/data/com.termux/files/usr/share/
drwx------ builder/4139      0 2017-11-27 16:49 ./data/data/com.termux/files/usr/share/man/
drwx------ builder/4139      0 2017-11-27 16:49 ./data/data/com.termux/files/usr/share/man/man1/
-rw------- builder/4139   4905 2017-11-27 16:49 ./data/data/com.termux/files/usr/share/man/man1/distccd.1.gz
-rw------- builder/4139   5300 2017-11-27 16:49 ./data/data/com.termux/files/usr/share/man/man1/include_server.1.gz
-rw------- builder/4139    888 2017-11-27 16:49 ./data/data/com.termux/files/usr/share/man/man1/distccmon-text.1.gz
-rw------- builder/4139   2149 2017-11-27 16:49 ./data/data/com.termux/files/usr/share/man/man1/pump.1.gz
-rw------- builder/4139  13325 2017-11-27 16:49 ./data/data/com.termux/files/usr/share/man/man1/distcc.1.gz
```